coralproject/talk
# Feat/CORL-2213 remove story live update config

## What does this PR do?
This PR removes the option to toggle live updates via the config panel in the stream.

## What changes to the GraphQL/Database Schema does this PR introduce?
None.

## How do I test this PR?
While logged in as an admin, navigate to a stream's Configure tab, and see that there is no longer the option to toggle live updates for that particular story (they can still be toggled site wide).

## How do we deploy this PR?
This shouldn't require any special considerations.
